scripts/linters/test_files/invalid_assert_raises.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Do we still need this file, considering we aren't having any lint test for this, right?